code/game/machinery/doors/door.dm

```diff
@@ -198,7 +198,26 @@
 			do_animate("deny")
 	return
 
+/obj/machinery/door/airlock/attack_generic(mob/user)
+	if (MUTATION_FERAL in user.mutations)
+		user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*2)
+		playsound(loc, damage_hitsound, 50, 1)
+		attack_animation(user)
+
+		if((MACHINE_IS_BROKEN(src)||!arePowerSystemsOn(src)) && density)
+			visible_message(SPAN_DANGER("\The [user] manages to pry \the [src] open!"))
+			open(1)
+		else
+			visible_message(SPAN_DANGER("\The [user] smashes into \the [src]!"))
+			damage_health(10)
+		return
+	..()
+
 /obj/machinery/door/attack_hand(mob/user)
+	if (MUTATION_FERAL in user.mutations)
+		attack_generic(user, 15)
+		return
```

Comment:
This isn't necessary? In the parent call done in the next line, this is essentially already done
https://github.com/UristMcStation/UristMcStation/blob/12e0ffe8681baa752a6e68a4b324eca269b0cf3a/code/game/machinery/_machines_base/machinery.dm#L237-L243

Reply:
Keep in mind, I just cherrypicked Bay directly aside from qdel -> drop stuff.

I assume this is a bypass of the early return in the parent.